web_src/js/components/DashboardRepoList.vue

```diff
@@ -354,7 +359,12 @@ export default defineComponent({
           <svg-icon name="octicon-plus"/>
         </a>
       </h4>
-      <div class="ui attached segment repos-search">
+      <div v-if="isLoading && !reposTotalCount" class="ui attached segment" :class="{'is-loading': isLoading}"/>
+      <div v-if="isRepoEmpty" class="ui attached segment empty-placeholder">
+        <svg-icon name="octicon-git-branch" :size="24" class-name="empty-placeholder-icon"/>
```

Comment:
Just use `class=`

Reply:
Excuse me, is there any problem with using it like this? There are many cases of such usage in the code repository.

Reply:
Take a look at `svg-icon (SvgIcon)` 's code

Reply:
I've seen it. Thanks for the reminder.
https://github.com/kerwin612/gitea/blob/c6322b93d075ea41105a4902506779b1ea13967d/web_src/js/svg.ts#L228-L231